portal/portal-service-impl/impl/src/java/org/sakaiproject/portal/service/PortalServiceImpl.java

```diff
@@ -183,18 +183,19 @@ public void update(Observable observable, Object o) {
 			}
 			case SiteService.EVENT_SITE_UNPUBLISH: {
 				try {
-					Site site = siteService.getSite(event.getContext());
+					final String siteId = (!StringUtils.isBlank(event.getContext())) ? event.getContext() : siteService.idFromSiteReference(event.getResource());
```

Comment:
superfluous parens

Reply:
isNotBlank is preferred over !isBlank

Reply:
I can revise this once the other loose end is resolved.